modules/apps/web-experience/export-import/export-import-service/src/main/java/com/liferay/exportimport/content/processor/base/BaseTextExportImportContentProcessor.java

```diff
@@ -1272,11 +1272,45 @@ else if (urlSBString.contains(
 				url = url.substring(groupFriendlyURL.length());
 			}
 
+			while (true) {
+				pos = url.indexOf(StringPool.SLASH, 1);
+
+				Group urlGroup = null;
+
+				if (pos == -1) {
+					break;
+				}
+				else {
+					String groupName = url.substring(1, pos);
+
+					groupFriendlyURL = StringPool.SLASH + groupName;
+
+					urlGroup = GroupLocalServiceUtil.fetchFriendlyURLGroup(
+						group.getCompanyId(), groupFriendlyURL);
+				}
+
+				if (urlGroup != null) {
+					group = urlGroup;
+					groupId = urlGroup.getGroupId();
+
+					url = url.substring(groupFriendlyURL.length());
+				}
+			}
+
+			if (Validator.isNull(url)) {
+				return;
```

Comment:
@robertoDiaz here instead of return shouldn't we call continue? return will step out of the while loop from the top and stops processing the rest of the content